test/unit/services/configBranchPlatform.test.js

```diff
@@ -1,6 +1,7 @@
+
+const originalPlatform = process.platform;
```

Comment:
this is weird... with the jest updates, using "this" no longer works... moved it to a variable instead